src/components/accomodations_card/index.js

```diff
@@ -0,0 +1,30 @@
+import EmojiPicker from "emoji-picker-react";
+
+const AccomodationsCard = () => {
+    return (
+        <div className='accomodations-card'>
+            <div className="accomodations-card">
+                <p><h3>Hotel Accomodations </h3></p>
+                <p> For your convenience, a block of rooms have been reserved at the following hotels. Please make reservations in advance to secure the group rate: </p>  
+
+                <p><h5> Homewood Suites by Hilton Carle Place </h5></p>
+                <p> 40 Westbury Ave Carle Place, NY 11514 </p>
+                <p> 📞 (516) 747-0230 </p>
+                <p> Click <a href= "www.my-event.hilton.com/allitawfikwedding2022/">here</a> to make reservations online or call and mention Alli/Tawfik Wedding before 8/24/2022 </p>
+                <p> **This hotel is nearby the wedding venue. </p>
```

Comment:
As al alternative to all the `<p>` tags you can wrap the entire block in a div with a class you can style

```suggestion
            <div className="accommodation_item">
                <h5> Homewood Suites by Hilton Carle Place </h5>
                <p> 40 Westbury Ave Carle Place, NY 11514 </p>
                <p> 📞 (516) 747-0230 </p>
                <p> Click <a href= "www.my-event.hilton.com/allitawfikwedding2022/">here</a> to make reservations online or call and mention Alli/Tawfik Wedding before 8/24/2022 </p>
                <p> **This hotel is nearby the wedding venue. </p>
            </div>
```